cluster/aws/config-test.sh

```diff
@@ -118,12 +117,6 @@ if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
   TARGET_NODE_UTILIZATION="${KUBE_TARGET_NODE_UTILIZATION:-0.7}"
 fi
 
-# Optional: Enable deployment experimental feature, not ready for production use.
-ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-false}"
```

Comment:
I think it is - I moved it in config-default.sh. Probably need to replicate those changes to config-test too.